src/main/java/study/step3/model/Movement.java

```diff
@@ -0,0 +1,22 @@
+package study.step3.model;
+
+import java.util.Random;
+
+/**
+ * Movement Class
+ *
+ * 이동을 제한하는 클래스
+ */
+public class Movement {
+    private static final int RANDOM_RANGE = 10;
+    private static final int MOVE_RESTRICT = 4;
+    private Random random;
+
+    public Movement() {
+        random = new Random();
+    }
+
+    public int stopAndMove() {
+        return random.nextInt(RANDOM_RANGE) >= MOVE_RESTRICT ? 1 : 0;
```

Comment:
```suggestion
        if(random.nextInt(RANDOM_RANGE) >= MOVE_RESTRICT) { 
               return 1;
            }
           return 0; 
        
```

잘 작성해주셨네요 :) 
저의 의견을 하나 드리자면  
early return pattern을 사용해보시는게 어떠신지 의견드립니다.
전달드린 url에도 작성되어있지만 early return pattern을 사용하시면 코드가 더 자연스럽게 읽혀질 것 같기도 해서 전달드립니다.

참고만 부탁드려요! 
https://jheloper.github.io/2019/06/write-early-return-code/

Reply:
링크까지 주셔서 감사합니다. :)